.asf.yaml

```diff
@@ -17,13 +17,23 @@
 
 github:
   description: "Apache Ranger - To enable, monitor and manage comprehensive data security across the Hadoop platform"
```

Comment:
Consider adding "and beyond" at end of the description.: "data security across the Hadoop platform and beyond"